### DISCUSSION TITLE
Is nng production-ready?

### DISCUSSION THREAD

**Discussion by pierre-rouleau** [nanomsg/nng#1862] [Q&A] [answered]
From what I've read so far, nng looks great!

Even though I like what I see, can I consider it production ready? 
 
Or is it still experimental as mentioned on https://nanomsg.org/gettingstarted/index.html ?

Should I consider nng as a candidate to replace other, older message passing sub-systems used in systems that exist today?

**Comment by gdamore** [comment, marked as answer]
Thanks for finding that reference.  NNG is *not* considered experimental.  I'll remove the reference that indicates otherwise.
